styles.css

```diff
@@ -3,21 +3,49 @@
 @import './button/button.css';
 @import './badges/badges.css';
 @import './cards/cards.css';
+@import './image/image.css';
+@import './input/input.css';
 
 @media (min-width: 750px) {
-    body{
-    max-width:750px;
+    main{
+    max-width:720px;
     }
 }
 
+img {
```

Comment:
do the same changes for img tag

Reply:
This is again for documentation site custom styles.css not a part of component library. This was intended to make all the images responsive.